packages/react-data-grid/style/rdg-core.less

```diff
@@ -17,4 +17,13 @@
   overflow-x: auto;
   overflow-y: scroll;
   background-color: #fff;
+  // will-change promotes the canvas to a new layer, improving scroll responsiveness
+  will-change: transform;
```

Comment:
This seems like a nuclear weapon. 🤔 

https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

Reply:
https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count

Reply:
Do we need `will-change` on both `canvas` and `rdg-rows-container`?